config/checkstyle-examples-checks.xml

```diff
@@ -115,6 +115,10 @@
       <property name="format" value="^class Example\d+"/>
       <property name="minimum" value="1"/>
       <property name="maximum" value="1"/>
+      <message key="regexp.minimum"
+               value="Example test input class name must match ''Example\d+''"/>
+      <message key="regexp.maximum"
+               value="Example test input must contain exactly one class matching ''Example\d+''"/>
```

Comment:
When converting the example to .java from .txt, I encountered this violation:
```
[ERROR] [checkstyle] [ERROR] /home/nick/IdeaProjects/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/finalclass/Example1.java:18: Line matches the illegal pattern '^class Example\d+'. [exampleClAuto-merging src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```

The issue was multiple classes named `ExampleN' (see the .txt file). This violation is sort of confusing at first, since I did have a class named `Example1`. I have updated the maximum message to be more descriptive.